### PR TITLE
Mrc 1991 - Hide map legends when no data to display

### PR DESCRIPTION
**Pull request by carbowizzy**
## Description

This PR hides legend control pane in maps when selections have not data to display. This PR doesn't hide indicator pane as it is useful for filtering prevalence even when there are not data to display on a particular prevalence.

## Type of version change

Minor

## Checklist

- [x] I have incremented version number, or version needs no increment
- [x] The build passed successfully, or failed because of ADR tests

### CODE REVIEW

File: src/app/static/src/app/hintVersion.ts [hunk 1]
```diff
@@ -1,2 +1,2 @@
 
-export const currentHintVersion = "1.1.3";
+export const currentHintVersion = "1.1.4";
```
Comment (EmmaLRussell) [inline src/app/static/src/app/hintVersion.ts:2]: I think we should get into the way of incrementing minor version (i.e. the second number) when we put in new behaviour changes which aren't just bug fixes, as described in the readme. It's maybe debatable which this is, but I think it's less of a bug fix and more of an improvement, which would make this 1.2.0

File: src/app/static/src/app/components/plots/choropleth/Choropleth.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@
                              :level-labels="featureLevels"
                              @detail-changed="onDetailChange"
                              @indicator-changed="onIndicatorChange"></map-control>
-                <map-legend :metadata="colorIndicator"
+                <map-legend v-show="!emptyFeature" :metadata="colorIndicator"
```
Comment (EmmaLRussell) [inline src/app/static/src/app/components/plots/choropleth/Choropleth.vue:24]: I'd probably just do do this as a v-if, no reason to render the control at all here.

**Review by EmmaLRussell — CHANGES_REQUESTED** [review]
Looks good, need to add it for the bubble plot too - both size legend and map legend (which is the colour legend).

File: src/app/static/src/app/components/plots/choropleth/Choropleth.vue (hunk 2 above)
Comment (carbowizzy) [reply src/app/static/src/app/components/plots/choropleth/Choropleth.vue:24]: The v-if behaviour was erratic on the map, freezing indicators control, while v-show happens to work well.

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**